testing/cypress/integration/restaurant_routing.spec.js

```diff
@@ -8,12 +8,12 @@ context('Actions', () => {
 
   describe('Navigate to Nandoz Page', function() {
     it('Home page loaded successful', function() {
-      cy.visit('http://192.168.86.34:3000');
+      cy.visit('localhost:3000');
       cy.contains('Footer');
     });
 
     it('Select Nandoz', function() {
-        cy.get('#root > div > div > div.LandingPage_tileContainer__1yaPd > div:nth-child(1)')
```

Comment:
May I know why you still need to change this css selector, still not working on your machine?